libs/commandline/README.md

```diff
@@ -52,7 +52,10 @@ Add the following dependency tag to the pom.xml of the dependant project:
 <dependency>
     <groupId>io.github.padaiyal.libs</groupId>
     <artifactId>commandline</artifactId>
+<<<<<<< HEAD
```

Comment:
Looks like this merge conflict needs to be resolved.

Reply:
Good catch! Resolved.

Reply:
Nice, thanks.